.github/.codecov.yml

```diff
@@ -1,3 +1,7 @@
 coverage:
-  round: up
-  precision: 2
+  round: down
+  precision: 1
+
+codecov:
+  notify:
+    wait_for_ci: true
```

Comment:
[This](https://docs.codecov.io/docs/common-recipe-list#send-coverage-notifications-even-if-all-ci-statuses-have-not-completed) will theoretically only run a codecov report once CI is green. We'll see :)

Reply:
No more premature +/ 8% changes in coverage? 😃

Reply:
@freddyaboulton I sure hope not 😆